src/actions/wallet/writeDepositETH.ts

```diff
@@ -23,7 +18,7 @@ export type WriteDepositETHParameters<
 > = Omit<
   WriteContractParameters<
     typeof l1StandardBridgeABI,
-    'writeDepositETH',
```

Comment:
this should probably have been breaking a test? Maybe write a test for it if not?

Reply:
I'd expect it to have raised a type error in the tests we already had :/ not sure how to write a test for it, as it's a type level error